Marlin/Configuration.h

```diff
@@ -921,6 +921,10 @@
 //#define MANUAL_Y_HOME_POS 0
 //#define MANUAL_Z_HOME_POS 0
 
+// If enabled, the global workspace offset will not include the Z home offset.
```

Comment:
// If enabled, the global workspace offset will not include the **_any_** home offset.